install_config/cluster_metrics.adoc

```diff
@@ -357,6 +357,12 @@ millicores) would request 4 CPUs.
 |`openshift_metrics_cassandra_storage_group`
 |The supplemental storage group to use for Cassandra.
 
+|`openshift_metrics_cassandra_nodeselector`
+|Set to the desired, existing
+xref:../admin_guide/scheduling/node_selector.adoc[node selector] to ensure that
```

Comment:
Need an anchor after the .adoc (same for all instances on the page). Everything else looks great 👍